### PR TITLE
docs: correct spelling of against

### PR DESCRIPTION
**Pull request by jlewis91**
**What this PR does / why we need it**: Fixes code comment typo

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: garden-service/src/config/project.ts [hunk 1]
```diff
@@ -285,7 +285,7 @@ export const projectDocsSchema = () =>
         .description((<any>environmentsSchema().describe().flags).description),
       providers: joiArray(providerConfigBaseSchema()).description(
         "A list of providers that should be used for this project, and their configuration. " +
-          "Please refer to individual plugins/providers for details on how to configure them."
+        "Please refer to individual plugins/providers for details on how to configure them."
```
Comment (edvald) [inline garden-service/src/config/project.ts]: The `prettier` linter is complaining about this

**Comment by jlewis91** [comment]
@edvald fixed. :)

**Comment by eysi09** [comment]
Looks like the `check-docs` script is failing. You need to run `npm run generate-docs` from the root to fix this and commit the changes.

Note that you need to have `parallel` installed to run the script. You can install with most package managers, for example on `macOS` you can do `brew install parallel`.

**Comment by jlewis91** [comment]
@eysi09 Thanks for the pointers. Checks pass now.

**Pull request merged**